js/fun-2/tests/unit.test.js

```diff
@@ -0,0 +1,56 @@
+import { handle } from '../index.js';
+import dbPoolConnection from "../connection-pool.js";
+
+jest.mock('../connection-pool.js', () => {
```

Comment:
@Mohmn what's the command to run this test

Reply:
@dadiorchen 
npm test

i forgot to update jest config in package.json, i just commited it